exercises/sum-of-multiples/sum_of_multiples_test.go

```diff
@@ -2,72 +2,37 @@ package summultiples
 
 import "testing"
 
-var test35 = []struct {
-	limit int
-	sum   int
-}{
-	{1, 0},
-	{4, 3},
-	{10, 23},
-	{100, 2318},
-	{1000, 233168},
-}
-
 var varTests = []struct {
 	divisors []int
 	limit    int
 	sum      int
 }{
+	{[]int{3, 5}, 1, 0},
+	{[]int{3, 5}, 4, 3},
+	{[]int{3, 5}, 10, 23},
+	{[]int{3, 5}, 100, 2318},
+	{[]int{3, 5}, 1000, 233168},
 	{[]int{7, 13, 17}, 20, 51},
 	{[]int{43, 47}, 10000, 2203160},
 	{[]int{5, 10, 12}, 10000, 13331672},
 	{[]int{1, 1}, 10000, 49995000},
-	// Note: The following test case deviates from the README.
-	// The README specifies some rather odd defaults, whereas
-	// this has the more logical approach of not implementing any
-	// defaults, which causes the resulting sum to be zero.
-	// See discussion in:
-	// https://github.com/exercism/xgo/issues/256 and
-	// https://github.com/exercism/x-common/issues/198
 	{[]int{}, 10000, 0},
 }
 
-func Test35(t *testing.T) {
-	sum35 := MultipleSummer(3, 5)
-	for _, test := range test35 {
-		s := sum35(test.limit)
-		if s != test.sum {
-			t.Fatalf("Sum to %d returned %d, want %d.", test.limit, s, test.sum)
-		}
-	}
-}
-
 func TestVar(t *testing.T) {
```

Comment:
With the removal of `Test35` we could rename this, since the `Var` is no longer needed to distinguish it from the `35`. I'm not sure that simply `Test` will be accepted, but maybe `TestSum` will suffice. Optional change.